packages/create-react-admin/templates/common/package.json

```diff
@@ -12,8 +12,8 @@
     "dependencies": {
         "@emotion/react": "^11.14.0",
         "@emotion/styled": "^11.14.0",
-        "@mui/icons-material": "^6.4.0",
-        "@mui/material": "^6.4.0",
+        "@mui/icons-material": "^7.0.1",
+        "@mui/material": "^7.0.1",
```

Comment:
One could argue we should also upgrade the minimum required version of `react-admin` to 5.8, but the latest version will be installed in any case, besides this would break the CI tests :grimacing:

Reply:
And it will be done by the release script ;)